server/services/licenceService.ts

```diff
@@ -426,6 +428,32 @@ export default class LicenceService {
     return compareLicenceConditions(originalLicence, variation)
   }
 
+  async getApprovalConversation(variation: Licence, user: User): Promise<ApprovalComment[]> {
+    const conversationEventTypes = [
+      LicenceEventType.VARIATION_SUBMITTED_REASON.valueOf(),
+      LicenceEventType.VARIATION_REFERRED.valueOf(),
+    ]
+
+    const licenceEvents = await this.licenceApiClient.matchLicenceEvents(
+      `${variation.id}`,
+      conversationEventTypes,
+      'eventTime',
+      'DESC',
+      user
+    )
+
+    const conversation = licenceEvents?.map(event => {
+      return {
+        who: `${event.forenames} ${event.surname}`,
+        when: event.eventTime,
+        comment: event.eventDescription,
+        role: event.eventType === LicenceEventType.VARIATION_SUBMITTED_REASON ? 'COM' : 'APPROVER',
```

Comment:
Some inference here but I think it works, only a COM can submit. Wondering if there are other events we will show on the conversation which the com does and which isn't submitting a variation